library/options.inc.php

```diff
@@ -3986,8 +3986,8 @@ function isOption($options, $test)
         }
     }
     $options = json_decode($options);
-    
-    return in_array($test, $options, true) ? true : false; // finally!
+
+    return !is_null($options) && in_array($test, $options, true) ? true : false; // finally!
```

Comment:
layout_options.edit_options should be defined as NOT NULL DEFAULT ''. Why would this ever be null? Perhaps your database has this wrong?

Reply:
I don't know.. right now when you save a 'edit demographics' screen  all the rows with empty edit_options are filled with string of 'null'.. and I have a lot of warnings in the patient file.
![image](https://user-images.githubusercontent.com/17809866/32037100-37471128-ba2c-11e7-92f0-d32a4a72c730.png)

Reply:
It prob gets set in legacy modifier test. I'll post a fix in next couple hour!